script/hassfest/translations.py

```diff
@@ -116,6 +116,19 @@ def translation_key_validator(value: str) -> str:
     return value
 
 
+def translation_value_validator(value: Any) -> str:
+    """Validate that the value is a valid translation.
+
+    - prevents string with HTML
+    - prevents combined translations
+    """
+    value = cv.string_with_no_html(value)
+    regex = re.compile(r"(.+\[%)|(%\].+)")
```

Comment:
This can be compiled once (as a constant outside of this method) and re-used every call.